language/stdlib/modules/LibraConfig.move

```diff
@@ -73,6 +73,33 @@ module LibraConfig {
         reconfigure_();
     }
 
+    // Set a config item to a new value and trigger a reconfiguration.
+    public fun set_with_capability<Config: copyable>(
```

Comment:
We might want this name to shout that it triggers reconfiguration to avoid surprises (`set_and_reconfigure`?)

Reply:
There is another `set` function that also triggers the reconfiguration, and I didn't touch it...
I'll rename this one to `set_with_capability_and_reconfigure` then...